addOns/network/src/main/resources/org/zaproxy/addon/network/resources/Messages.properties

```diff
@@ -0,0 +1,2 @@
+network.ext.name = Network Extension
+network.ext.desc = Provides networking capabilities.
```

Comment:
Do we want to say "Provides core networking capabilities." here so that it's more clear that websockets or whatever future tech might be on its own?

Reply:
Updated in all places.